src/guides/v2.2/release-notes/release-notes-2-2-11-commerce.md

```diff
@@ -14,7 +14,7 @@ Magento 2.2.11 has not been tested with PHP 7.1. PHP 7.1 reached EOL (End of Lif
 
 Twenty-three security enhancements that help close cross-site scripting (XSS) and remote code execution (RCE) vulnerabilities as well as other security issues. No confirmed attacks related to these issues have occurred to date. However, certain vulnerabilities can potentially be exploited to access customer information or take over administrator sessions. Most of these issues require that an attacker first obtains access to the Admin. As a result, we remind you to take all necessary steps to protect your Admin, including but not limited to these efforts: IP whitelisting, two-factor authentication, use of a VPN, the use of a unique location rather than /admin, and good password hygiene.
 
-With this quarterly release, we’ve changed how we describe these security issues.  Individual issues are no longer described in the Magento Security Center. Instead, these issues are documented in an Adobe Security bulletin. Please see [Security updates available for Magento | APSB20-02](https://helpx.adobe.com/security/products/magento/apsb20-02.html) for more information.
+With this quarterly release, we’ve changed how we describe these security issues. Individual issues are no longer described in the Magento Security Center. Instead, these issues are documented in an Adobe Security bulletin. Please see [Security updates available for Magento | APSB20-02](https://helpx.adobe.com/security/products/magento/apsb20-02.html) for more information.
```

Comment:
This paragraph still has problems.

Reply:
Oops. Well, since Jenkins is almost finished and this is only a minor space issue, I'm going to punt on it.

Reply:
It's not a minor space problem. It cause a weird shift in the paragraph that makes it look like a table gone wrong. I'll get a screenshot for you....

Reply:
you're right. that needs to be fixed. 

@jfrontain, FYI for future reference. Spaces around pipes are important.

Reply:
<img width="1178" alt="Screen Shot 2020-01-27 at 5 05 31 PM" src="https://user-images.githubusercontent.com/13662379/73221766-4bd4f900-4127-11ea-86dd-dc545d7a090b.png">